typings/index.d.ts

```diff
@@ -0,0 +1,133 @@
+import {
+    CSSProperties,
+    CSSPropertiesArray,
+    CSSPropertiesComplete,
+    CSSPropertiesLossy,
+    CSSPropertiesPseudo,
+    CSSPropertiesRecursive,
+    CSSWideKeyword,
+} from './css-properties';
+
+export {
+    CSSProperties,
+    CSSPropertiesArray,
+    CSSPropertiesComplete,
+    CSSPropertiesLossy,
+    CSSPropertiesPseudo,
+    CSSPropertiesRecursive,
+    CSSWideKeyword,
+};
+
+/**
+ * Aphrodite style declaration
+ */
+export type StyleDeclarationMap = Map<keyof CSSProperties, string | number>;
+export type StyleDeclaration<T = {}> = {
+    [P in keyof T]: CSSProperties | StyleDeclarationMap
+};
+
+/**
+ * Return value from StyleSheet.create.
+ */
+export interface StyleDeclarationValue<
+    T extends CSSProperties | StyleDeclarationMap = {}
+> {
+    _len: number;
+    _name: string;
+    _definition: T;
+}
+
+interface StyleSheetStatic {
+    /**
+     * Create style sheet
+     */
+    create<T extends StyleDeclaration<T>>(
+        styles: T
+    ): { [K in keyof T]: StyleDeclarationValue<T[K]> };
+    /**
+     * Rehydrate class names from server renderer
+     */
+    rehydrate(renderedClassNames: string[]): void;
+}
+
+export var StyleSheet: StyleSheetStatic;
+
+type CSSInputTypes = StyleDeclarationValue | false | null | void;

```

Comment:
I think you can either do ` | false | void` or do ` | false | null | undefined`. I believe `void` is equivalent to `null | undefined`.

Reply:
I actually had to put null back in.  While doing tests null was not explicitly allowed, and failed the `css(null)` test